misc-scripts/schema_patcher.pl

```diff
@@ -280,12 +281,13 @@ sub about {
 
 # Get available patches.
 
-foreach my $thing ( [ 'ensembl',                 'core',       'table.sql'  ],
-                    [ 'ensembl-compara',         'compara',    'table.sql'  ],
-                    [ 'ensembl-funcgen',         'funcgen',    'table.sql'  ],
-                    [ 'ensembl-variation',       'variation',  'table.sql'  ],
-                    [ 'ensembl-production',      'production', 'table.sql'  ],
-                    [ 'ensembl-ontology-schema', 'ontology',   'tables.sql' ] )
+foreach my $thing ( [ 'ensembl',                 'core',           'table.sql'  ],
+                    [ 'ensembl-compara',         'compara',        'table.sql'  ],
+                    [ 'ensembl-funcgen',         'funcgen',        'table.sql'  ],
+		    [ 'ensembl-gene2phenotype',  'gene2phenotype', 'table.sql'  ],
```

Comment:
This line has tabs instead of spaces. Although there are other, decade-old tabs in this file, I feel that new code should adhere to the coding standards

Reply:
Changed TABS with spaces and cleaned up commit history